cpp/models/abm/location.h

```diff
@@ -245,6 +246,21 @@ class Location
         m_capacity_adapted_transmission_risk = consider_capacity;
     }
 
+    /**
+     * Save the current number of individuals in the each infection state at a specific time step
+     * @param t the time step to save
+    */
+    void add_subpopulations_to_time_series(const TimePoint& t);
+
+    /**
+     * Return the time series object of the current number of individuals in the each infection state
+     * @return the time series object of the current number of individuals in the each infection state
+    */
+    const TimeSeries<double>& get_time_series_subpopulations() const
```

Comment:
Why not simply `get_population`?

Reply:
There's another get_population function in location to count the number of persons at the location. Maybe you can suggest another name?